xrpl/asyncio/transaction/main.py

```diff
@@ -1,17 +1,19 @@
 """High-level transaction methods with XRPL transactions."""
+
 import math
 from typing import Any, Dict, Optional, cast
 
 from typing_extensions import Final
 
 from xrpl.asyncio.account import get_next_valid_seq_number
 from xrpl.asyncio.clients import Client, XRPLRequestFailureException
+from xrpl.asyncio.clients.client import _get_network_id_and_build_version
```

Comment:
Any function that gets imported elsewhere shouldn't start with a `_`.

Reply:
fixed in ab918dc